host/usb/idf_component.yml

```diff
@@ -3,7 +3,7 @@ description: USB Host library
 version: "1.0.0-rc0"
 url: https://github.com/espressif/esp-usb/tree/master/host/usb
 dependencies:
-  idf: ">=6.0"
+  idf: ">=5.4"
 targets:
   - esp32s2
   - esp32s3
```

Comment:
Unrelated to this MR, but so I don't forget:
Let's add esp32h4 here too, before v1 release